stable/coredns/templates/configmap.yaml

```diff
@@ -12,6 +12,9 @@ metadata:
     kubernetes.io/name: "CoreDNS"
     {{- end }}
     app.kubernetes.io/name: {{ template "coredns.name" . }}
```

Comment:
Can we do this change in separate PR ? Mixing them will make it harder to revert if we for some reason need to go back.

Reply:
sure, will do.